src/__tests__/components/auth/__test__/homeComponent.test.js

```diff
@@ -0,0 +1,10 @@
+import { shallow } from "enzyme";
+import React from "react";
+import Home from "../../../../components/auth/homeComponent";
+
+describe("Home component", () => {
```

Comment:
Add a better description to this `describe`

Reply:
Changed the description to home view component